slackify-html.js

```diff
@@ -147,9 +146,25 @@ function walk(dom, nesting) {
             }
             break;
           case 'h1':
+            content = walk(el.children);
```

Comment:
oh you can just put the code under the h4 case i think and it will work for h1,h2,h3,h4 instead of repeating the code.